kiya.go

```diff
@@ -56,9 +55,8 @@ func Move(kmsService *cloudkms.Service,
 	if err := PutSecret(kmsService, storageService, target, targetKey, sourceValue); err != nil {
 		return tre.New(err, "save key failed", targetKey)
 	}
-	fmt.Printf("Successfully copied [%s] to [%s]\n", sourceKey, target.Label)
 	// delete key from source
-	if err := DeleteSecret(kmsService, storageService, target, targetKey); err != nil {
+	if err := DeleteSecret(kmsService, storageService, source, sourceKey); err != nil {
```

Comment:
strange change, target->source.   r u sure?

Reply:
Given that this is in the "move" command, we fist "put" the secret in the new space (e.g. copy), then we need to delete the origin (source), making this a "move"